capella_console_client/session.py

```diff
@@ -159,8 +159,8 @@ def _cache_user_info(self):
     def _set_api_key_auth_header(self, api_key: Optional[str]):
         assert isinstance(api_key, str)
         api_key = api_key.strip()
-        if not api_key.startswith("Apikey"):
-            api_key = f"Apikey {api_key}"
+        if not api_key.lower().startswith("apikey"):
+            api_key = f"ApiKey {api_key}"
```

Comment:
Is the capital **K** in  Api**K**ey intentional?

Reply:
(I realize that we are case insensitive in our authorizer but think we should define a standard)

Reply:
> Is the capital **K** in Api**K**ey intentional?

Yes. This is the standard this is being used in the target API code (even though it is not case sensitive).